### PR TITLE
CAMEL-11099: Resolve issue caused by unresolve elements hanging off t…

### PR DESCRIPTION
**Pull request by linead**
Details @ https://issues.apache.org/jira/browse/CAMEL-11099

### CODE REVIEW

File: components/camel-soap/src/main/java/org/apache/camel/dataformat/soap/Soap11DataFormatAdapter.java [hunk 1]
```diff
@@ -199,8 +199,18 @@ private Exception createExceptionFromFault(Fault fault) {
                 throw new RuntimeCamelException(e);
             }
         }
+
+	Object detailObj = faultDetail.getAny().get(0);
```
Comment (zregvart) [inline components/camel-soap/src/main/java/org/apache/camel/dataformat/soap/Soap11DataFormatAdapter.java]: There's a tab character here, we have a spaces policy, you can run source checks from Maven:

    $ cd components/camel-soap; mvn -Psourcecheck clean verify

This is what I get now:

    ...
    [INFO] --- maven-checkstyle-plugin:2.17:checkstyle (validate) @ camel-soap ---
    [INFO] Starting audit...
    [ERROR] /home/zregvart/workspaces/camel/camel/components/camel-soap/src/main/java/org/apache/camel/dataformat/soap/Soap11DataFormatAdapter.java:203:1: File contains tab characters (this is the first instance). [FileTabCharacter]
    Audit done.
    ...

File: components/camel-soap/src/main/java/org/apache/camel/dataformat/soap/Soap11DataFormatAdapter.java [hunk 2]
```diff
@@ -199,8 +199,18 @@ private Exception createExceptionFromFault(Fault fault) {
...
+
+        if (!JAXBIntrospector.isElement(detailObj)) {
```
Comment (zregvart) [inline components/camel-soap/src/main/java/org/apache/camel/dataformat/soap/Soap11DataFormatAdapter.java]: This does not compile, `JAXBIntrospector.isElement` cannot be referenced as a static method, see [javadoc](https://docs.oracle.com/javaee/7/api/javax/xml/bind/JAXBIntrospector.html#isElement-java.lang.Object-).

Comment (linead) [inline components/camel-soap/src/main/java/org/apache/camel/dataformat/soap/Soap11DataFormatAdapter.java]: Sorry - wrote this in a rush this morning and found that method. Will clean it up

**Comment by linead** [comment]
Thanks @zregvart cleaned this up

**Comment by zregvart** [comment]
Thank you for your contribution 👍, this has been merged to master you can close this pull request

**Pull request closed without merging**